pkg/interface/soto/src/js/api.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import _ from 'lodash';
+
+class UrbitApi {
```

Comment:
Outside of scope for this PR, but isn't this duplicated in practically everything under `pkg/interface`? What happened to `urb.js`?

Reply:
We don’t use urb.js anymore and specify API files for each application. We may break these out into npm packages eventually, but we aren’t right now.